### PR TITLE
move to metadata.json & bump version

### PR DESCRIPTION
**Pull request by mmoll**
closes GH-5

maybe the authors should be updated?

### CODE REVIEW

**Comment by ekohl** [comment]
I'm not sure. I think a majority of the code was still written by onyx point but we do maintain it.

File: metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name":         "theforeman-concat_native",
+  "version":      "1.3.2",
+  "author":       "Onyx Point, Inc.",
+  "summary":      "Native type for multi-part file concatenatenation",
```
Comment (domcleal) [inline metadata.json]: You've concatenated a few too many letters here ;-)

**Comment by domcleal** [comment]
I'm happy leaving Onyx Point as the author, or appending us to it.  We should probably clear up the README at some point too.  It's our most popular Forge module IIRC!

File: metadata.json (hunk 1 above)
Comment (mmoll) [inline metadata.json]: damn ;)

**Comment by mmoll** [comment]
I did set the author to "theforeman" now, I don't know if that field is supporting an array, the name doesn't suggest that.

File: metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name":         "theforeman-concat_native",
+  "version":      "1.3.2",
+  "author":       "theforeman",
+  "summary":      "Native type for multi-part file concatenation",
+  "license":      "Apache-2.0",
+  "source":       "git://github.com/theforeman/puppet-concat",
+  "project_page": "https://github.com/theforeman/puppet-concat",
```
Comment (domcleal) [inline metadata.json]: Should we rename the repo to "puppet-concat_native"?  It just confused a few folks on #puppet-dev, so now would be a good time if we did.

Comment (ekohl) [inline metadata.json]: :+1: we already do so on the forge and when installing.

Comment (domcleal) [inline metadata.json]: Done, would you mind updating this @mmoll?  I think it's ready then.

**Comment by mmoll** [comment]
updated.

**Comment by ekohl** [comment]
:+1:

**Comment by domcleal** [comment]
Thanks @mmoll!  I'll release it too.

**Pull request merged**